pyk/src/pyk/kcfg.py

```diff
@@ -295,12 +296,21 @@ def pretty_print(self, kprint: KPrint) -> List[str]:
         def _bold(text: str) -> str:
             return '\033[1m' + text + '\033[0m'
 
+        def _green(text: str) -> str:
```

Comment:
What do you think about extracting `pretty_print` to a module or class of its own?

Reply:
I'll add this to the TODO for the pretty_print PR